apps/protocol-api/src/main.ts

```diff
@@ -93,7 +93,7 @@ prisma.$use(async (params, next) => {
       chain_id: contributionParams.chain_id,
     });
   }
-  if ((params.model == 'Attestation', params.action === 'createMany')) {
```

Comment:
great catch. what was this evaluating to beforehand? what is the truth value of (bool, bool)?